src/compiler/compile/css/Selector.ts

```diff
@@ -147,7 +147,7 @@ export default class Selector {
 		for (const block of this.blocks) {
 			for (const selector of block.selectors) {
 				if (selector.type === 'PseudoClassSelector' && selector.name === 'global') {
-					if (/[^\\],/.test(selector.children[0].value)) {
+					if (/[^\\],(?![^([]+(?:\)|\]))/.test(selector.children[0].value)) {
```

Comment:
it is possible to escape the `)` character in the selector:

```css
:global(.abc, .def\)) {
}
```

I propose update the regex to handle it:

```js
/[^\\],(?!([^([]+[^\\]|[^([\\])[)\]])/
```

Reply:
I might be missing some other cases, but I've added another test case for the escaped parentheses, and the current pattern seem to handle it fine. Is there any other case where it would fail with the current regex?

Reply:
The example that I showed above, non-escaped comma with escaped bracket

```css
:global(.abc, .def\)) {
}
```

Reply:
I'm not sure how I missed that in my test. Updated, thanks!

Reply:
no worries 👍🏻